utils/scripts/flash-jtag.cfg

```diff
@@ -0,0 +1,17 @@
+init
+reset halt
+targets 0
+clk32M 4000
+gdb_breakpoint_override hard
+
+source utils/scripts/partition.conf
+load_image   $arc_bin   $arc_start_addr
+load_image   $x86_bin   $quark_start_addr
```

Comment:
These throw error unless both `$arc_bin` and `$x86_bin` are defined. We need to `if` them out, or duplicate these with the variants, and use `if` from bash